src/TraceEvent/TraceEvent.cs

```diff
@@ -2903,6 +2904,11 @@ private void ConfirmAllEventsAreInEnumeration()
                 declaredSet.Add(eventName, eventName);
             }
 
+            // These events are derived from GCDynamic and has no template associated with them
+            declaredSet.Remove("CommittedUsage");
+            declaredSet.Remove("HeapCountTuning");
```

Comment:
TODO: Double check this.

Reply:
Reasonable to do this.  Can you put this under a check to only do it if the parser is the CLR parser?

Reply:
Done - worth adding that I checked if these are from the GCDynamicEventParser.